modules/aerodyn/src/BEMT.f90

```diff
@@ -1452,9 +1475,12 @@ subroutine BEMT_CalcOutput_Inductions( InputIndex, t, CalculateDBEMTInputs, Appl
          !............................................
          ! get BEMT inductions (axInduction and tanInduction):
          !............................................
-         call calculate_Inductions_from_BEMT(p, phi, u, OtherState, AFInfo, axInduction, tanInduction, ErrStat2, ErrMsg2)
+         call calculate_Inductions_from_BEMT(p, phi, u, OtherState, AFInfo, axInduction, tanInduction, ErrStat2, ErrMsg2, k_out, kp_out, F_out)
```

Comment:
Will we run into compatibility issues with older compilers passing optional arguments like this to another subroutine?

Reply:
Thanks. Yeah, I was feeling shaky while implementing this. Right now I don't really have another way of doing it.... So... fingers crossed?

Reply:
Since those arguments are optional in `calculate_Inductions_from_BEMT`, you could do something like:
`If (present(k_out)) then
   call calculate_Inductions_from_BEMT(p, phi, u, OtherState, AFInfo, axInduction, tanInduction, ErrStat2, ErrMsg2, k_out, kp_out, F_out)
else
             call calculate_Inductions_from_BEMT(p, phi, u, OtherState, AFInfo, axInduction, tanInduction, ErrStat2, ErrMsg2)
endif`

or something similar.  From what I could tell, it looks like you have either none or all of those arguments in the parent routine.

Reply:
That's a great suggestion, I've now updated the code.